src/System/Wlog/LogHandler.hs

```diff
@@ -63,8 +63,8 @@ class LogHandler a where
 
     -- | Logs an event if it meets the requirements
     -- given by the most recent call to 'setLevel'.
-    handle :: MonadIO m => a -> LogRecord -> LoggerName -> m ()
-    handle h lr@(LR pri _) logname =
+    logHandlerMessage :: MonadIO m => a -> LogRecord -> LoggerName -> m ()
+    logHandlerMessage h lr@(LR pri _) logname =
```

Comment:
This function should be moved out of this type class. It has default implementation. And nobody uses other implementation. This is very confusing. So this functions should be moved to top-level out from this type class. If somebody one day will need to override this implementation we can always move back to this type class.